lib/utils.js

```diff
@@ -7,6 +7,12 @@ function isPlainObject(value) {
 }
 exports.isPlainObject = isPlainObject;
 
+// Support dynamic require within webpack bundle.
+if (typeof __non_webpack_require__ === 'undefined') {
+  var __non_webpack_require__ = require;
```

Comment:
Ah nice! @benjamn because you're take is assigning it directly t `exports.require` it gets transformed into:
```js

// Support dynamic require within webpack bundle.
exports.require =  false
  ? require : require;
```
which works! I had run into issues where webpack would create a local undefined var of `require` in some cases.